tests/metrics/functional/test_sequence.py

```diff
@@ -0,0 +1,206 @@
+import pytest
+from pytorch_lightning.metrics.functional.sequence import bleu_score
+from nltk.translate.bleu_score import sentence_bleu, corpus_bleu, SmoothingFunction
+
+# https://www.nltk.org/api/nltk.translate.html?highlight=bleu%20score#nltk.translate.bleu_score.SmoothingFunction
+smooth_func = SmoothingFunction().method2
+
+# example taken from https://www.nltk.org/api/nltk.translate.html?highlight=bleu%20score#nltk.translate.bleu_score.sentence_bleu
+hypothesis1 = "It is a guide to action which ensures that the military always obeys the commands of the party".split(' ')
+hypothesis2 = "It is to insure the troops forever hearing the activity guidebook that party direct".split(' ')
+reference1 = "It is a guide to action that ensures that the military will forever heed Party commands".split(' ')
+reference2 = "It is the guiding principle which guarantees the military forces always being under the command of the Party".split(' ')
+reference3 = "It is the practical guide for the army always to heed the directions of the party".split(' ')
+
+
+def test_with_sentence_bleu():
+    nltk_output = sentence_bleu([reference1, reference2, reference3], hypothesis1, weights=(1, 0, 0, 0))
```

Comment:
this looks very similar, you are testing the same just with different `weights` and `n`, right?
if so use parameterization so it is easier to see that it is the same...

Reply:
Yea, I have parameterized it.

Should I also parametrize `smooth` argument?
Right now, I am testing with 4 different `weights` and `n` with `smooth=True` and `smooth=False`.

Reply:
yes, I missed the `smooth` :rabbit: